### ISSUE TITLE
Modpack Server crashes

### ISSUE THREAD

**Issue by SylverPlayer** [Teneted/Tenet#2878]
**Minecraft Version :** 1.20.1

**Mohist Version :** 1.20.1-515

**Operating System :** Windows 10

**Logs :** (https://pastebin.com/xgKsVAM3)

**Mod list :** Modpack BMC4

**Plugin list :** Ninguno de momento

**Description of issue :** El servidor con el modpack BMC4 crashea antes de llegar a abrirse el servidor

**Comment by killer2gudpl** [comment]
Try removing ysns, should help. @SylverPlayer

**Comment by SylverPlayer** [comment]
ysns?

**Comment by killer2gudpl** [comment]
> ysns?

Yes, there is a mod installed called YSNS in your minecraft. Could be called You Shall Not Spawn or YSNS.

**Comment by SylverPlayer** [comment]
I was trying to open it with forge. Now i'm trying to use mohist and the log changed into this one. I removed the YSNS too.

https://pastebin.com/fmT6ZV8p

**Comment by MA0B** [comment]
same error, here i tryied everithing

**Comment by MA0B** [comment]
@SylverPlayer u fix that?
